### ISSUE TITLE
Problem with handles in tutorial

### ISSUE THREAD

**Issue by ephraimtabackman** [sghall/react-compound-slider#19]
### Problem or feature description:

I am following the tutorial. At the second step where we add handles, `<Slider.Handles>` children function receives an object whose `handles` property is `undefined`. Therefore, calling `handles.map` throws an error.

### Steps to reproduce (for problems):

```
<Slider
    rootStyle={sliderStyle}
    domain={[0, 100]}
    step={1}
    mode={2}
    values={[30]}
  >
    <div style={railStyle} />
    <Slider.Handles>
      {({ handles, getHandleProps }) => (
// handles is undefined here
        <div className="slider-handles">
          {handles.map(handle => (
            <Handle
              key={handle.id}
              handle={handle}
              getHandleProps={getHandleProps}
            />
          ))}
        </div>
      )}
    </Slider.Handles>
  </Slider>
```

### Versions (for problems):

React-Compound-Slider:
^0.11.0

React:
^16.3.2

Browser:
Chrome 66.0.3359.170

Operating System:
Windows 10 (1709)

**Comment by sghall** [comment]
Hey.   I couldn't actually recreate your exact issue, but this really had me going.  There's some garbage in the tutorial that I need to clean up. Thanks for pointing this out.   I think the slider code is fine, it's just the tutorial.

Here's a sandbox with your test code working https://codesandbox.io/s/3o2n7qjym

**Comment by reicheltp** [comment]
Hi, I have the same problem using `react-compound-slider` with nextjs. I dug a little deeper and discovered that the type check at https://github.com/sghall/react-compound-slider/blob/master/src/Slider/Slider.js#L369-L374 fails. See: 

![image](https://user-images.githubusercontent.com/4380869/40471494-28f4c7d6-5f37-11e8-8f93-1be576fd83fc.png)

See https://github.com/reicheltp/react-compound-slider-nextjs for reproduction

**Comment by sghall** [comment]
Alrighty.  Released this fix in 0.12.0.  I was never able to reproduce the issue, but I'm going to embrace the mystery.  If this doesn't actually solve the issue for anyone please open this back up.
